scripts/gitlab/publish-docs.sh

```diff
@@ -7,6 +7,7 @@ clone_repos() {
     echo "__________Clone repos__________"
     git clone https://github.com/parity-js/jsonrpc.git jsonrpc
     git clone https://github.com/paritytech/wiki.git wiki
+    git clone https://github.com/paritytech/parity-config-generator
```

Comment:
I'd make sure these repos do not exist before cloning
```suggestion
    rm -rf jsonrpc wiki paruty-config-generator
    git clone https://github.com/paritytech/parity-config-generator
```

Reply:
This script shouldn't be running in an image that the script has already run in before, IIUC? This change is only useful for testing this script?